mono/metadata/w32file-unix.c

```diff
@@ -14,6 +14,12 @@
 #ifdef HAVE_SYS_STATVFS_H
 #include <sys/statvfs.h>
 #endif
+#ifdef HAVE_COPYFILE_H
+#include <copyfile.h>
+#  if !defined(COPYFILE_CLONE)
+#    #define COPYFILE_CLONE 0
```

Comment:
why is this needed?

Reply:
Older MacOS versions do not have COPYFILE_CLONE

Reply:
Can you unconditionally define it to the correct value?
Or you want 0 (incorrect value?) as no-op on old systems?

Reply:
Correct value is #define COPYFILE_CLONE          (1<<24)
but intent might be to make it a no-op downlevel.

Reply:
Good idea